workflows/proteomics/clinicalmp/clinicalmp-verification/iwc_clinicalmp_verification_workflow_tests.yml

```
@@ -0,0 +1,49 @@
+- doc: Test outline for Clinical Metaproteomics Verification Workflow
+  job:
+    SGPS peptide report:
+      class: File
+      path: test-data/SGPS_peptide-report.tabular
+      filetype: tabular
+    Distinct Peptides for PepQuery:
+      class: File
+      path: test-data/Distinct_Peptides_for_PepQuery.tabular
+      filetype: tabular
+    MaxQuant peptide report:
+      class: File
+      path: test-data/MaxQuant-peptide-report.tabular
+      filetype: tabular
+    Input MGFs:
```

```suggestion
    Tandem Mass Spectrometry (MS/MS) datasets:
```
The user interface specifies that these need to be MGF files